odoo/addons/base/tests/test_image.py

```diff
@@ -314,3 +314,10 @@ def _orientation_test(self, orientation, colors, size, expected):
         self._assertAlmostEqualSequence(fixed_image.getpixel((size - 1, 0)), expected[1])         # top/right
         self._assertAlmostEqualSequence(fixed_image.getpixel((0, size - 1)), expected[2])         # bottom/left
         self._assertAlmostEqualSequence(fixed_image.getpixel((size - 1, size - 1)), expected[3])  # bottom/right
+
+    def test_ptype_image_to_jpeg(self):
+        """Test that P type image convert to JPEG"""
+        desired_output = 2968636781
+        image = Image.new('P', (10, 10), color='red')
+        converted_output = binascii.crc32(tools.image.image_apply_opt(image, 'JPEG'))
+        self.assertEqual(desired_output, converted_output)
```

Comment:
Do we really need to hash and hardcode the result like this?
Can't we make 2 images, one P and one RGB and compare they are equal at the end?

This `desired_output` doesn't really have any meaning.